RunScripts/test_en-de.sh

```diff
@@ -0,0 +1,5 @@
+cp /projects/unmt/communication-translation/Data/translation_references/de-en.* Output/en-de_pipeline
```

Comment:
If I understand correctly, this file can be reverted since there is an existing [test_en-de_pipeline.sh](https://github.com/CLMBRs/communication-translation/blob/main/RunScripts/test_en-de_pipeline.sh) which does the same thing